CMakeLists.txt

```diff
@@ -137,4 +137,10 @@ if (NOT_SUBPROJECT)
       DESTINATION
         ${RAPIDFUZZ_CMAKE_CONFIG_DESTINATION}
     )
-endif(NOT_SUBPROJECT)
\ No newline at end of file
+    
+    set(CPACK_PACKAGE_VENDOR "Max Bachmann")
+    set(CPACK_PACKAGE_CONTACT "kontakt@maxbachmann.de")
```

Comment:
I would prefer to use the project url here, so e.g. issues are reported via the issue tracker.
```suggestion
    set(CPACK_PACKAGE_CONTACT "https://github.com/maxbachmann/rapidfuzz-cpp")
```

Reply:
I think you may be thinking of CPACK_PACKAGE_HOMEPAGE_URL.  From what I can see in the cpack docs, the "contact" is an email address.